package.json

```diff
@@ -1,55 +1,57 @@
 {
-  "name": "validator",
+  "name": "@blockr/blockr-validator",
   "version": "1.0.0",
-  "description": "blockchain-validator",
-  "main": "src/server.js",
-  "dependencies": {
-    "@sentry/node": "^4.6.4",
-    "async": "^2.6.1",
-    "bip39": "^2.5.0",
-    "bluckur-database": "git+https://github.com/TimoF97/blox-database.git",
-    "bluckur-models": "git+https://github.com/Bluckur/bluckur-models.git",
-    "core-js": "^2.5.7",
-    "dotenv": "^5.0.1",
-    "express": "^4.16.3",
-    "hashmap": "^2.3.0",
-    "js-nacl": "^1.2.2",
-    "level": "^3.0.2",
-    "node-cron": "^1.2.1",
-    "node-random-name": "^1.0.1",
-    "opn": "^5.3.0",
-    "qrcode": "^1.2.0",
-    "readline-sync": "^1.4.9",
-    "request": "^2.87.0",
-    "seedrandom": "^2.4.3",
-    "sleep": "^6.0.0",
-    "smokesignal": "^0.2.2",
-    "socket.io": "^2.1.1",
-    "socket.io-client": "^2.1.1",
-    "uuid": "^3.2.1"
-  },
-  "devDependencies": {
-    "eslint": "^4.19.1",
-    "eslint-config-airbnb-base": "^12.1.0",
-    "eslint-plugin-import": "^2.12.0"
-  },
+  "description": "Blockr validator package",
+  "main": "dist/server.js",
+  "types": "dist/server.d.ts",
   "scripts": {
-    "start": "node src/server",
+    "build": "tsc -p tsconfig.json",
+    "build-watch": "tsc -w -p tsconfig.json",
     "test": "echo \"Error: no test specified\" && exit 1",
```

Comment:
Change message. There are no tests, so no test can be specified

Reply:
This is the default generated message, but yes when we add tests this will be changed.